u2flib_server/__init__.py

```diff
@@ -25,4 +25,4 @@
 # ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
 # POSSIBILITY OF SUCH DAMAGE.
 
-__version__ = "3.2.1-dev"
+__version__ = "3.3.0-dev"
```

Comment:
I don't have a problem with that at all. I figured I'd let you guys make the call on that. I'll increment the version to 4.0.0-dev